tracer/test/Datadog.Trace.TestHelpers/EnvironmentTools.cs

```diff
@@ -109,7 +109,7 @@ public static string GetTracerTargetFrameworkDirectory()
 #elif NETCOREAPP3_1_OR_GREATER
             return "netcoreapp3.1";
 #elif NETCOREAPP || NETSTANDARD
```

Comment:
Remove this?
```suggestion
#elif NETCOREAPP
```

Reply:
or maybe even?
```suggestion
#elif NETCOREAPP2_1_OR_GREATER
```

Reply:
Done 👍 https://github.com/DataDog/dd-trace-dotnet/pull/5260/commits/8362a6f134756d23c3596a3e4c6904d59145cb88